tcms/testruns/templates/testruns/get.html

```diff
@@ -0,0 +1,102 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+
+{% block title %}TR-{{ test_run.pk }}: {{ test_run.summary }}{% endblock %}
```

Comment:
Use `object` instead of `test_run`. If not available then modify the backend context to pass the object variable.